docs/transaction.md

```diff
@@ -149,7 +149,7 @@ These are the current default values in the dashcore library involved on these c
 - `Transaction.DUST_AMOUNT`: `546` (duffs)
 
 ## Fee calculation
-When outputs' value don't sum up to the same amount that inputs, the difference in dashes goes to the miner of the block that includes this transaction. The concept of a "change address" usually is associated with this: an output with an address that can be spent by the creator of the transaction.
+When outputs' value don't sum up to the same amount that inputs, the difference in funds goes to the miner of the block that includes this transaction. The concept of a "change address" usually is associated with this: an output with an address that can be spent by the creator of the transaction.
```

Comment:
I might use "value" instead of "funds" here, or just remove "in funds" altogether. But maybe I would rewrite the first sentence like this:

When the sum of the outputs' value doesn't add up to the sum of the inputs' value, the difference is given to the miner of the block that includes this transaction as a "mining fee".

Reply:
What about this ? : 

`When the sum of the outputs' value is less than the sum of the inputs' value, the difference is given to the miner of the block that includes this transaction as a "mining fee".`